diagrams.md

```diff
@@ -361,14 +360,6 @@ erDiagram
         String description
         Integer order
     }
-    GFE-OVERVIEW {
```

Comment:
Thanks for maintaining the diagram!

Reply:
Also need to remove relationship near the bottom

`ACQUISITION-PACKAGE ||--|| GFE-OVERVIEW : "TODO delete table"`